pkgs/applications/misc/milu/default.nix

```diff
@@ -27,13 +27,14 @@ stdenv.mkDerivation rec {
      glib
      unzip
      clang
+     gcc
```

Comment:
It uses `gcc`, but links with `libclang`. I am not entirely sure why it fails on darwin, but the error I got on the hydra ` https://hydra.nixos.org/build/35464414` said gcc not found. Hence I added it.